pom.xml

```diff
@@ -5,14 +5,13 @@
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>mate-academy</groupId>
-    <artifactId>oop-example</artifactId>
+    <artifactId>read-from-file</artifactId>
     <version>1.0-SNAPSHOT</version>
 
     <properties>
         <jdk.version>17</jdk.version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <maven.checkstyle.plugin.version>3.1.1</maven.checkstyle.plugin.version>
```

Comment:
Why did you removed it? Please revert this change in order to make maven checkstyle plugin to work properly. :)